packages/containers/src/List/List.connect.js

```diff
@@ -22,6 +26,15 @@ function getItems(state, config) {
 	return [];
 }
 
+function getPagedItems(state, config) {
+	const items = configureGetPagedItems(config)(state);
+
+	if (items) {
+		return items.toJS();
```

Comment:
you should never do a toJS in mapStateToProps.
This should only done during a rendering.